src/test/resources/application-test.yml

```diff
@@ -0,0 +1,2 @@
+logger:
+  config: logback-promtail.xml
```

Comment:
you can set this in the test (property annotation), no need to change the logging for all tests